test/src/device-request-prompt.spec.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2023 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import expect from 'expect';
+import {TimeoutError} from 'puppeteer';
+
+import {launch} from './mocha-utils.js';
+
+describe('device request prompt', function () {
+  let testState: Awaited<ReturnType<typeof launch>>;
+
+  beforeEach(async () => {
```

Comment:
Nit: Let's set it up the browser as we do for `ignorehttpserrors.spec.ts`
that way we if we add test they will use the same browser instance.

Reply:
Done